Documentation/user-guides/alerting.md

```diff
@@ -30,9 +32,72 @@ spec:
   replicas: 3
 ```
 
-The Alertmanager instances will not be able to start up, unless a valid configuration is given. The following example configuration sends notifications against a non-existent `webhook`, allowing the Alertmanager to start up, without issuing any notifications.
+The Alertmanager instances will not be able to start up, unless a valid configuration is given. To create a configuration, AlertmanagerConfig resource or manully created secret could be used.
 For more information on configuring Alertmanager, see the Prometheus [Alerting Configuration document][alerting-config].
 
+## AlertmanagerConfig Resource
+
+The following example configuration creates an AlertmanagerConfig resource that sends notifications against a non-existent `wechat` receiver, allowing the Alertmanager to start up, without issuing any notifications.
```

Comment:
```suggestion
The following example configuration creates an AlertmanagerConfig resource that sends notifications to a non-existent `wechat` receiver:
```

(removing the part of the sentence that suggests that something like this would be necessary to allow Alertmanager to start)